apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/AIPolicyChat.tsx

```diff
@@ -74,21 +85,53 @@ export const AIPolicyChat = ({
         Make sure to verify any generated code or suggestions, and share feedback so that we can
         learn and improve.`}
         >
-          <Button
-            type="default"
-            className="w-min"
-            icon={
-              <div
-                className={cn(
-                  'w-2 h-2 rounded-full',
-                  includeSchemaMetadata ? 'bg-brand' : 'border border-stronger'
-                )}
-              />
-            }
-            onClick={() => snap.setShowAiSettingsModal(true)}
-          >
-            {includeSchemaMetadata ? 'Include' : 'Exclude'} database metadata in queries
-          </Button>
+          {includeSchemaMetadata ? (
+            <SchemaComboBox
+              className="w-fit"
+              disabled={!includeSchemaMetadata}
+              selectedSchemas={selectedSchemas}
+              onSelectSchemas={setSelectedSchemas}
+              label={
+                includeSchemaMetadata && selectedSchemas.length > 0
+                  ? `${selectedSchemas.length} schema${
+                      selectedSchemas.length > 1 ? 's' : ''
+                    } selected`
+                  : 'No schemas selected'
+              }
+            />
+          ) : (
+            <Tooltip_Shadcn_>
```

Comment:
We now have a `ButtonTooltip` component to handle this one 🙂

Reply:
although might need tweaking - just noticed TooltipContent here will need to be JSX